config/bidderinfo_test.go

```diff
@@ -448,18 +429,16 @@ func TestProcessAliasBidderInfo(t *testing.T) {
 					AliasOf: "bidderA",
 				},
 			},
-			expectError: "bidderA not found for an alias: bidderB",
+			assertFunc: func(bidderInfos BidderInfos, err error) {
+				assert.Nil(t, bidderInfos)
+				assert.Equal(t, "bidder: bidderA not found for an alias: bidderB", err.Error())
+			},
 		},
 	}
 
 	for _, test := range testCases {
 		bidderInfos, err := processBidderAliases(test.aliasInfos, test.bidderInfos)
-		if test.expectError != "" {
-			assert.ErrorContains(t, err, test.expectError)
-		} else {
-			assert.Equal(t, test.expectedBidderInfos, bidderInfos)
-		}
-
+		test.assertFunc(bidderInfos, err)
```

Comment:
If each test has it's own assertion there isn't much value in using a table driven format. If there is no good way to generalize the assertions, please use a series of `t.Run` calls in this method instead.

Reply:
https://github.com/prebid/prebid-server/pull/3001/commits/89c536348e8a1c08462358ec6bae92a2bf71b1bc updates to remove `assertFunc` and use common assertion